components/popover/style.scss

```diff
@@ -3,94 +3,107 @@
 	z-index: z-index( ".components-popover" );
 	left: 50%;
 
-	&:before {
-		border: 8px solid $light-gray-500;
-	}
-
-	&:after {
-		border: 8px solid $white;
-	}
-
-	&:before,
-	&:after {
-		content: "";
-		position: absolute;
-		margin-left: -10px;
-		height: 0;
-		width: 0;
-		border-left-color: transparent;
-		border-right-color: transparent;
-		line-height: 0;
-	}
-
-	&.is-top {
-		bottom: 100%;
-		margin-top: -8px;
-
+	&:not(.is-mobile) {
 		&:before {
-			bottom: -8px;
+			border: 8px solid $light-gray-500;
 		}
 
 		&:after {
-			bottom: -6px;
+			border: 8px solid $white;
 		}
 
 		&:before,
 		&:after {
-			border-top-style: solid;
-			border-bottom: none;
+			content: "";
+			position: absolute;
+			margin-left: -10px;
+			height: 0;
+			width: 0;
+			border-left-color: transparent;
+			border-right-color: transparent;
+			line-height: 0;
 		}
-	}
 
-	&.is-bottom {
-		top: 100%;
-		margin-top: 8px;
+		&.is-top {
+			bottom: 100%;
+			margin-top: -8px;
 
-		&:before {
-			top: -8px;
-		}
+			&:before {
+				bottom: -8px;
+			}
 
-		&:after {
-			top: -6px;
+			&:after {
+				bottom: -6px;
+			}
+
+			&:before,
+			&:after {
+				border-top-style: solid;
+				border-bottom: none;
+			}
 		}
 
-		&:before,
-		&:after {
-			border-bottom-style: solid;
-			border-top: none;
+		&.is-bottom {
+			top: 100%;
+			margin-top: 8px;
+
+			&:before {
+				top: -8px;
+			}
+
+			&:after {
+				top: -6px;
+			}
+
+			&:before,
+			&:after {
+				border-bottom-style: solid;
+				border-top: none;
+			}
 		}
 	}
 }
 
 .components-popover__content {
-	position: absolute;
 	box-shadow: $shadow-popover;
 	border: 1px solid $light-gray-500;
 	background: $white;
-	min-width: 240px;
+	height: 100%;
 
 	&:focus {
 		box-shadow: $button-focus-style;
 	}
 
-	.components-popover.is-top & {
+	.components-popover:not(.is-mobile) & {
+		position: absolute;
+		min-width: 240px;
+		height: auto;
+	}
+
+	.components-popover:not(.is-mobile).is-top & {
 		bottom: 100%;
 	}
 
-	.components-popover.is-center & {
+	.components-popover:not(.is-mobile).is-center & {
```

Comment:
Instead of using a series of :not(.is-mobile) and given that is the component itself that sets the is-mobile depending on a width. I think using a CSS rule that uses our variable $break-medium may be an option to simplify this.

Reply:
`is-mobile` can be disabled even on mobile, because this behavior is opt-out. Relying on the $break-medium would apply this consistently.